README.md

```diff
@@ -30,8 +30,12 @@ S3_DIR_PATH=<base-path-in-bucket> \
 
 ## Test
 
+You must have a local DC/OS CLI installed and configured against a
+running DC/OS cluster.
+
 ```
-make test
+$ cd tests
+$ source env/bin/activate
```

Comment:
@mgummelt this is not available by default. should I run cli/bin/env.sh or create my own?